js/jsonparser.js

```diff
@@ -26,6 +28,7 @@ function updateVertretungsplan() {
           //plan has to be set visible before calling updatePlan() because
           //calculating the correct offset for the plan wouldn't work otherwise
           showById('plan');
+          json = responseJSON;
```

Comment:
Du kannst dem Button auch immer eine neue Funktion zuweisen.
irgendwo in der Funktion machst du:

    //docuement.getelemen....
    button.onclick = function () { //...